tip-wlan/charts/opensync-gw-static/templates/deployment.yaml

```diff
@@ -47,7 +47,12 @@ spec:
             failureThreshold: {{ .Values.probes.readinessProbe.failureThreshold }}
             periodSeconds: {{ .Values.probes.readinessProbe.periodSeconds }}
             successThreshold: {{ .Values.probes.readinessProbe.successThreshold }}
-          {{- end }}       
+          startupProbe:
+            tcpSocket:
+              port: {{ .Values.service.port2 }}
```

Comment:
Can we have a more descriptive names than `port1` or `port2` ?

Reply:
no idea what these are for. we certainly could, probably not in this pr?

Reply:
Agree with @4c74356b41. It's something we should definitely fix but not in this PR.